content/security/advisory/2024-01-24.adoc

```diff
@@ -122,14 +122,14 @@ issues:
     ====
 
     **Fix Description:** +
-    Jenkins 2.442, LTS 2.426.3 disables the command parser feature that replaces an `@` character followed by a file path in an argument with the file's contents for CLI commands.
+    Jenkins 2.442, LTS 2.426.3, and LTS 2.440.1 disable the command parser feature that replaces an `@` character followed by a file path in an argument with the file's contents for CLI commands.
```

Comment:
Please keep the singular. If we wanted this to be plural (and there's a case to be made for that!), we'd already have it in the original advisory.